app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragment.kt

```diff
@@ -7,10 +7,24 @@ import android.view.View
 import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
 import org.oppia.app.model.PromotedStory
+import org.oppia.app.story.StoryFragment
 import javax.inject.Inject
 
+const val KEY_INTERNAL_PROFILE_ID = "INTERNAL_PROFILE_ID"
+
 /** Fragment that contains all recently played stories. */
 class RecentlyPlayedFragment : InjectableFragment(), OngoingStoryClickListener {
+  companion object {
+    /** Returns a new [StoryFragment] to display the story corresponding to the specified story ID. */
```

Comment:
Update this comment accordingly.

Reply:
Updated to :`/** Returns a new [RecentlyPlayedFragment] to display recently played stories. */`

Reply:
/** Returns a new [RecentlyPlayedFragment] to display recently played stories. */